src/os/im/action/abstractimportaction.js

```diff
@@ -3,7 +3,9 @@ goog.module.declareLegacyNamespace();
 
 const osXml = goog.require('os.xml');
 
-const IImportAction = goog.requireType('os.im.action.IImportAction');
+// The compiler has to process this first or @inheritDoc will not work properly on implementing classes.
+// @see https://github.com/google/closure-compiler/issues/3583
+const IImportAction = goog.require('os.im.action.IImportAction'); // eslint-disable-line opensphere/no-unused-vars
```

Comment:
Is importing this and assigning the return value with a lint disable a better solution than just a naked require? I know we can't use naked requires with ES6 modules, but we aren't on those yet and disabling the linter is kinda ugly.

Reply:
In this particular case, you need an assignment for the `const IImportAction` referenced in the `@implements` tag so you can't just use a bare `require`.

For the broader issue like requires for directives only referenced in the template, I've gone back and forth on what to do. You can still call `goog.require` in an ES module, but Closure will warn you if you're doing that with another ES module. We will have to use an `import` statement with an `eslint-disable-line` comment at that point, so I don't think it would hurt to start doing that now. I'm fine with either approach though for goog modules.